From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for deleted snd_hda_register_beep_device()
Patch-mainline: Never, kABI fix for SLE15
References: bsc#1122944

Just add a dummy function for the deleted and not used
snd_hda_register_beep_device() for kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/pci/hda/hda_beep.c |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/sound/pci/hda/hda_beep.c
+++ b/sound/pci/hda/hda_beep.c
@@ -339,3 +339,10 @@ int snd_hda_mixer_amp_switch_put_beep(st
 	return snd_hda_mixer_amp_switch_put(kcontrol, ucontrol);
 }
 EXPORT_SYMBOL_GPL(snd_hda_mixer_amp_switch_put_beep);
+
+/* only for kABI compatibility */
+int snd_hda_register_beep_device(struct hda_codec *codec)
+{
+	return 0;
+}
+EXPORT_SYMBOL_GPL(snd_hda_register_beep_device);
